extensions/azurecore/src/utils.ts

```diff
@@ -183,10 +185,13 @@ export function updateCustomCloudProviderSettings(defaultSettings: ProviderSetti
 				Logger.info(`Custom provider settings loaded for ${cloudProvider.settings.metadata.displayName}`);
 			}
 			void vscode.window.showInformationMessage(localize('providerSettings.success', 'Successfully loaded custom endpoints from settings'));
+			TelemetryReporter.sendTelemetryEvent('loadCustomEndpointsSuccess');
 
 		} catch (error) {
 			void vscode.window.showErrorMessage(localize('providerSettings.error', 'Could not load endpoints from settings, please check the logs for more details.'));
 			console.error(error.message);
+			TelemetryReporter.sendTelemetryEvent('loadCustomEndpointsError');
+			TelemetryReporter.sendErrorEvent2(TelemetryViews.AzureCore, 'loadCustomEndpointsError', error.message)
```

Comment:
FYI, this won't be sending anything unless `includeMessage` is true. 

https://github.com/microsoft/ads-extension-telemetry/blob/main/src/telemetryReporter.ts#L342

1. Make sure you're actually verifying that the events being sent up contain the info you expect (using Fiddler)
2. Before setting includeMessage to true - will the message possibly contain any PII? For example, if a parse error occurs will it include possibly-sensitive content from the custom endpoint itself?

Reply:
I think it could potentially include sensitive content, in that case should I just stick with the initial `TelemetryReporter.sendTelemetryEvent('loadCustomEndpointsError');`?

Reply:
If you pass in the entire error object it will remove the message but keep the stack trace, which could at least provide some helpful information to get started with. Then if you need more we can sit down later and try to find a solution that works from a privacy standpoint too.